infra/helper.py

```diff
@@ -410,18 +410,20 @@ def build_image(args):
     print('Using cached base images...')
 
   # If build_image is called explicitly, don't use cache.
-  if _build_image(args.project_name, no_cache=True, pull=pull):
+  if build_image_impl(args.project_name, no_cache=True, pull=pull):
     return 0
 
   return 1
 
 
-def build_fuzzers(args):
+def build_fuzzers(args, skipbuild_image_impl=False):
```

Comment:
add an underscore between "skip" and "build" and drop `impl`. I don' think the parameter should depend on the function name so heavily.

Reply:
Why is this parameter even needed by the way?